### PR TITLE
Readme update to add images of contributors

### PR DESCRIPTION
**Pull request by tamzi**
## Summary

This adds an automated way to show all contributors profiles in the readme of the project

## How It Was Tested

Preview the readme

## Screenshot/Gif

image

### CODE REVIEW

**Comment by mariobodemann** [comment]
Needs to be `ZeContributors` - header!!! /jk, maybe.

**Review by mariobodemann — APPROVED** [review]
I approve. (Even though the contribution should be higher up in the readme ... maybe.)

File: README.md [hunk 1]
```diff
@@ -98,3 +98,10 @@ Badge --> Badge: Show user page `u`
 Alice -->> Badge: Button 'down'
 Badge --> Badge: Show user page `d`
 ```
+### Contributors
```
Comment (alisen) [inline README.md:101]:
```suggestion
### ZeContributors
```

**Review by alisen — COMMENTED** [review]
one suggesion

**Review by alisen — APPROVED** [review]

**Pull request merged**